webapp/src/main/java/com/box/l10n/mojito/utils/StatsDMetricsConfig.java

```diff
@@ -0,0 +1,94 @@
+package com.box.l10n.mojito.utils;
+
+import org.springframework.boot.actuate.autoconfigure.metrics.MeterRegistryCustomizer;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.lang.Nullable;
+
+/**
+ * StatsD metrics related config
+ */
+@Component
+@ConfigurationProperties("l10n.management.metrics.export.statsd")
+@ConditionalOnProperty(value="management.metrics.export.statsd.enabled", havingValue="true")
+public class StatsDMetricsConfig {
+
+    private NamingTransformation nameTransformation = NamingTransformation.NONE;
+
+    private String tagKeyPrefix = "";
+
+    private String tagValuePrefix = "";
+
+    private String namePrefix = "";
+
+    public NamingTransformation getNameTransformation() {
+        return nameTransformation;
+    }
+
+    public void setNameTransformation(NamingTransformation nameTransformation) {
+        this.nameTransformation = nameTransformation;
+    }
+
+    public String getTagKeyPrefix() {
+        return tagKeyPrefix;
+    }
+
+    public void setTagKeyPrefix(String tagKeyPrefix) {
+        this.tagKeyPrefix = tagKeyPrefix;
+    }
+
+    public String getTagValuePrefix() {
+        return tagValuePrefix;
+    }
+
+    public void setTagValuePrefix(String tagValuePrefix) {
+        this.tagValuePrefix = tagValuePrefix;
+    }
+
+    public String getNamePrefix() {
+        return namePrefix;
+    }
+
+    public void setNamePrefix(String namePrefix) {
+        this.namePrefix = namePrefix;
+    }
+
+    public NamingConvention buildNamingConvention() {
+        return new NamingConvention() {
+
+            @Override
+            public String name(String name, Meter.Type type, @Nullable String baseUnit) {
+                return namePrefix + nameTransformation.transform(name);
+            }
+
+            @Override
+            public String tagKey(String key) {
+                return tagKeyPrefix + nameTransformation.transform(key);
+            }
+
+            @Override
+            public String tagValue(String value) {
+                return tagValuePrefix + nameTransformation.transform(value);
+            }
+        };
+    }
+
+    @Bean
+    @ConditionalOnProperty(value="l10n.management.metrics.export.statsd.transformName", havingValue="true")
+    MeterRegistryCustomizer<MeterRegistry> meterRegistryCustomizer() {
```

Comment:
to target statds only registry: `MeterRegistryCustomizer<StatsdMeterRegistry>`? (I fear this would apply to all backends)